sdkit/train/merge_models.py

```diff
@@ -1,47 +1,69 @@
-# loosely inspired by https://github.com/lodimasq/batch-checkpoint-merger/blob/master/batch_checkpoint_merger/main.py#L71
+"""
+Merges two models using weighted sum.
+
+ loosely inspired by:
+  - https://github.com/lodimasq/batch-checkpoint-merger/blob/master/batch_checkpoint_merger/main.py#L71
+"""
 
 from sdkit.utils import load_tensor_file, save_tensor_file, log
 
-def merge_models(model0_path: str, model1_path: str, ratio: float, out_path: str, use_fp16=True):
+
+def merge_models(
+    model0_path: str,
+    model1_path: str,
+    ratio: float,
+    out_path: str,
+    use_fp16=True
+):
     '''
     Merges (using weighted sum) and writes to the `out_path`.
 
     * model0, model1 - the first and second model files to be merged
-    * ratio - the ratio of the second model. 1 means only the second model will be used.
-              0 means only the first model will be used.
+    * ratio - the ratio of the second model.
+        1 means only the second model will be used.
+        0 means only the first model will be used.
     '''
 
-    log.info(f'[cyan]Merge models:[/cyan] Merging {model0_path} and {model1_path}, ratio {ratio}')
+    log.info('[cyan]Merge models:[/cyan] Merging %s and %s, ratio %s', 
+             model0_path, model1_path, ratio)
     merged = merge_two_models(model0_path, model1_path, ratio, use_fp16)
 
-    log.info(f'[cyan]Merge models:[/cyan] ... saving as {out_path}')
+    log.info('[cyan]Merge models:[/cyan] ... saving as %s', out_path)
 
     if out_path.lower().endswith(".safetensors"):
-        # elldrethSOg4060Mix_v10.ckpt contains a state_dict key among all the tensors, but safetensors
+        # elldrethSOg4060Mix_v10.ckpt contains a state_dict
+        #  key among all the tensors, but safetensors
         # assumes that all entries are tensors, not dicts => remove the key
         if 'state_dict' in merged:
-           del merged['state_dict']
+            del merged['state_dict']
         save_tensor_file(merged, out_path)
     else:
-        save_tensor_file({'state_dict':merged}, out_path)
+        save_tensor_file({'state_dict': merged}, out_path)
+
 
 # do this pair-wise, to avoid having to load all the models into memory
 def merge_two_models(model0, model1, alpha, use_fp16=True):
```

Comment:
No types on the model?

Reply:
The doc-string can mention that it's either a Tensor or a string (depicting the path to the model file).